lib/graph.py

```diff
@@ -135,11 +149,12 @@ def iter_jobs(self):
     def build(self):
         for job in self.iter_jobs():
             args = job["wrapper_arguments"]
-            cmd_label = args.get("description", args["command"])
-            if not cmd_label:
-                cmd_label = args["command"]
+            description = args["description"]
```

Comment:
These variables are only used once. I would just write `args["description"]` in the function call.